cmd/transportRequestUploadSOLMAN.go

```diff
@@ -5,16 +5,14 @@ import (
 	"github.com/SAP/jenkins-library/pkg/log"
 	"github.com/SAP/jenkins-library/pkg/piperutils"
 	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/SAP/jenkins-library/pkg/transportrequest"
 	"github.com/SAP/jenkins-library/pkg/transportrequest/solman"
```

Comment:
Can this not be removed now that "github.com/SAP/jenkins-library/pkg/transportrequest" is imported?

Reply:
No, the import is used by  `type transportRequestUploadSOLMANUtils interface `, see below.